src/Uno.UI/UI/Xaml/ApplicationHelper.cs

```diff
@@ -30,6 +30,8 @@ public static string RequestedCustomTheme
 						Application.Current.RequestedTheme = ApplicationTheme.Light;
 					}
 				}
+
+				Application.UpdateRequestedThemesForResources();
```

Comment:
Is this related to perf?

Reply:
Yes it is, it avoids the resources resolution to compute the active theme.